config/title_screens/README.txt

```diff
@@ -1,7 +1,10 @@
-The enclosed /images folder holds the image files used as the title screen for the game. All common formats such as PNG, JPG, and GIF are supported.
-Byond's DMI format is also supported, but if you use a DMI only include one image per file and do not give it an icon_state (the text label below the image).
+The enclosed /images folder holds the image files used as the title screen for the game.
+
+Specification:
+Formats: PNG, JPG, GIF, DMI*
+Dimensions: 480x544
```

Comment:
what?

Reply:
I didn't sleep well aslt night...

Reply:
as for 480^2

Sure if you like letter boxing....